src/chat/components/button-bar-ai-popover/index.tsx

```diff
@@ -83,86 +79,72 @@ const ButtonBarAIPopover: FC<Props> = ({
     }
   }, [activeOption, optionsData]);
 
+
   return (
+    <Popover open={open} onOpenChange={setOpen}>
     <div ref={rootRef}>
-      <ButtonBarAI
-        icon={icon}
-        aria-describedby={id}
-        withDropdownIcon={true}
-        tooltip={tooltip}
-        btnText={shortOptionName}
-        onClick={handleClick}
-      />
+      <PopoverTrigger asChild>
+        <div>
+          <ButtonBarAI
+            icon={icon}
+            aria-describedby={'simple-popover'}
+            withDropdownIcon={true}
+            tooltip={tooltip}
+            btnText={shortOptionName}
+            onClick={handleClick}
+          />
+        </div>
 
-      {optionsData.type === 'text' && (
-        <Popover
-          id={id}
-          className='bar-ai-popover-full'
-          open={isOpen}
-          anchorEl={anchorEl}
-          onClose={handleClose}
-          classes={{ paper: '-mt-2' }}
-          anchorOrigin={{
-            horizontal: 'center',
-            vertical: 'top',
-          }}
-          transformOrigin={{
-            horizontal: 'center',
-            vertical: 'bottom',
-          }}
-          closeAfterTransition={true}
-        >
-          <div className='bg-white p-2'>
-            {optionsData.options.map((option) => (
-              <div
-                className='text-ch-text-content flex w-full min-w-52 cursor-pointer justify-between px-1.5 py-1 text-sm tracking-wide'
-                key={option.value}
-                onClick={() => handleOptionChange(option.value)}
-              >
-                <div>{option.name}</div>
-                {activeOption === option.value && (
-                  <IconCheck className='text-ch-icon-blue h-4 w-4 flex-shrink-0' />
-                )}
-              </div>
-            ))}
+      </PopoverTrigger>
+      <PopoverContent
```

Comment:
popover's border and box shadow do not match the design

![image](https://github.com/user-attachments/assets/0bdd2bf9-6a8c-4071-9c74-14423eec7636)